### PR TITLE
New package: Google.Picasa.2 version 2

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/116537&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/g/Google/Picasa/2/2/Google.Picasa.2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.Picasa.2
+PackageVersion: 2
+Installers:
+- InstallerUrl: http://www.google.com/downloads/GoogleSoftwareDownloads.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (Trenly) [inline manifests/g/Google/Picasa/2/2/Google.Picasa.2.installer.yaml:10]:
```suggestion
  NestedInstallerType: nullsoft
```

File: manifests/g/Google/Picasa/2/2/Google.Picasa.2.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Google.Picasa.2
+PackageVersion: 2
```
Comment (Trenly) [inline manifests/g/Google/Picasa/2/2/Google.Picasa.2.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0
```

File: manifests/g/Google/Picasa/2/2/Google.Picasa.2.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Google.Picasa.2
+PackageVersion: 2
```
Comment (Trenly) [inline manifests/g/Google/Picasa/2/2/Google.Picasa.2.yaml:5]:
```suggestion
PackageVersion: 2.0
```

File: manifests/g/Google/Picasa/2/2/Google.Picasa.2.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Google.Picasa.2
+PackageVersion: 2
+PackageLocale: en-US
+Publisher: Google
```
Comment (Trenly) [inline manifests/g/Google/Picasa/2/2/Google.Picasa.2.locale.en-US.yaml:7]:
```suggestion
Publisher: Google, Inc.
```

File: manifests/g/Google/Picasa/2/2/Google.Picasa.2.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.Picasa.2
+PackageVersion: 2
```
Comment (Trenly) [inline manifests/g/Google/Picasa/2/2/Google.Picasa.2.installer.yaml:5]:
```suggestion
PackageVersion: 2.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
> Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\g\Google\Picasa\2\2', expected 'manifests\g\Google\Picasa\2\2.0'

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Pull request closed without merging**